aces/Loris
# [JSX] Added Modal Window Component - HACKTOBERFEST 2018 🎃

### Summary of changes
Added a Modal Window Component to LORIS.

### Why?
Convenience! This allows developers to integrate forms right into the page, essentially providing a visual portal for the users to interact in without having to navigate anywhere. This presents itself as a more elegant solution than using a bootstrap modal window because it's fully 🌈REACT🌈 and can be customized in the future if we so desire!

### Features
- Content and Title are completely 🌠customizable🌠
- Option of throwing warning upon Modal closure
- Optional passing of save function for Modals that include forms

### Testing
- A use case for the Modal window has been implemented in the Media Module.
- Go to the Media module menu filter. Click the `Edit` button on any of the datatable rows.
- Make sure everything is running smoothely!

![mw](https://user-images.githubusercontent.com/7550451/46967367-52e50e80-d07e-11e8-85e9-ee72a539a4d1.png)
